### PR TITLE
Add seedData functionality

### LINKED ISSUE #9: As a user exploring the app, I can populate the app with sample data
So that I can easily see how the app will look like when it is in use

### PR DESCRIPTION
**Pull request by kokerinks**
Closes #9

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SeedDataCommand.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                new Address("Blk 45 Aljunied Street 85, #11-31"),
+                new HashSet<>(Set.of(new Tag("colleagues")))
+        )
+    };
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        AddressBook addressBook = new AddressBook(model.getAddressBook());
+        boolean hasAdded = false;
+        for (Person person : sampleData) {
```
Comment (billyhoce) [inline src/main/java/seedu/address/logic/commands/SeedDataCommand.java:74]: Very good logic here that accounts for all cases (none, one, all persons already added)

File: src/test/java/seedu/address/logic/commands/SeedDataCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            new Email("royb@example.com"),
+            new Address("Blk 45 Aljunied Street 85, #11-31"),
+            new HashSet<>(Set.of(new Tag("colleagues")))
+        );
+    @Test
+    public void execute_emptyAddressBook_success() {
+        Model model = new ModelManager();
+        Model expectedModel = model;
+
+        assertCommandSuccess(new SeedDataCommand(), model, SeedDataCommand.MESSAGE_SUCCESS, expectedModel);
```
Comment (billyhoce) [inline src/test/java/seedu/address/logic/commands/SeedDataCommandTest.java:79]: Not very sure if this works the way I think it does, but doesn't this test case always pass as `expectedModel` and `model` is always referring to the same `Model` instance? Would it be more correct if you instanciate `expectedModel` to be a new Model containing a new `AddressBook` with the seed persons manually added into it?

File: src/test/java/seedu/address/logic/commands/SeedDataCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+    @Test
+    public void execute_addressBookWithSomeContacts_success() {
+        Model model = new ModelManager();
+        model.addPerson(ALICE);
+        model.addPerson(BERNICE);
+        model.addPerson(DAVID);
+        Model expectedModel = model;
+
+        assertCommandSuccess(new SeedDataCommand(), model, SeedDataCommand.MESSAGE_SUCCESS, expectedModel);
```
Comment (billyhoce) [inline src/test/java/seedu/address/logic/commands/SeedDataCommandTest.java:90]: See above comment

File: src/test/java/seedu/address/logic/commands/SeedDataCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        Model model = new ModelManager();
+        model.addPerson(ALICE);
+        model.addPerson(BERNICE);
+        model.addPerson(CHARLOTTE);
+        model.addPerson(DAVID);
+        model.addPerson(IRFAN);
+        model.addPerson(ROY);
+        Model expectedModel = model;
+
+        assertCommandSuccess(new SeedDataCommand(), model, SeedDataCommand.MESSAGE_FAILURE, expectedModel);
```
Comment (billyhoce) [inline src/test/java/seedu/address/logic/commands/SeedDataCommandTest.java:104]: See above comment

**Review by billyhoce — CHANGES_REQUESTED** [review]
Implementation looks good! But test cases might not be as complete, after fixing that, it should be all good!

**Comment by billyhoce** [comment]
Also, you might want to consider using `main/java/seedu.address/model/util/SampleDataUtil`

**Review by billyhoce — APPROVED** [review]

**Comment by kinjalagarwal1810** [comment]
Looks good!

**Pull request merged**